### PR TITLE
[mdbtools] Initial integration

### PR DESCRIPTION
**Pull request by evanmiller**
Project was previously approved in #4403.

### CODE REVIEW

File: projects/mdbtools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Evan Miller
```
Comment (jonathanmetzman) [inline projects/mdbtools/Dockerfile:1]: Needs to be "Google LLC"

File: projects/mdbtools/build.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Evan Miller
```
Comment (jonathanmetzman) [inline projects/mdbtools/build.sh:1]: Please fix copyright.

File: projects/mdbtools/build.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#!/bin/bash -eu
```
Comment (jonathanmetzman) [inline projects/mdbtools/build.sh:17]: I think the shebang line should be first.

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]

**Comment by evanmiller** [comment]
@jonathanmetzman Made the requested changes, let me know if you need anything else!

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**